### PR TITLE
Add preliminary schedule (data file)

### PR DESCRIPTION
**Pull request by mslw**
This PR adds a handcrafted Perntabarf-like XML schedule file compatible with Giggity.

My proposal is that now a few people should view the schedule through Giggitty and use this PR to make adjustments - without taking too much time before publishing, because Giggity handles updates rather seamlessly (unless you are tweaking the file and want an update *right* now, but that might be Github's caching, not Giggity's). Once published, we can continue tweaking the XML file by separate PRs.

For now, Giggity can be pointed to a Github raw URL: https://raw.githubusercontent.com/distribits/distribits-2024-website/schedule/static/sched/distribits2024.xml

When merged, the URL to use will be `https://distribits.live/sched/distribits2024.xml` (because of how hugo treats files in `static` directory).

### Pentabarf XML, are you serious?

I chose Pentabarf-XML over ical / xcal motivated by the following comment from Giggity's source code that said it is pretty excellent ([here](https://github.com/Wilm0r/giggity/blob/c3d7487e17e6e406aefc51572b28b62d5a016da7/app/src/main/java/net/gaast/giggity/Schedule.java#L809-L813) - since Giggity was written primarily to handle files produced by pentabarf and frab, I couldn't find any documentation to tell me which fields are necessary):

> Pentabarf, the old conference organisation tool has a pretty excellent native XML format and is now the preferred file format. http://pentabarf.org/Main_Page
> It's not really maintained anymore though, a recent fork called Frab is more maintained and Giggity can read its XML exports just as well https://github.com/frab/frab

But mainly because Giggity will parse speakers from pentabarf files, but not ical / xcal (because "iCal format is abused as a conference description format, but not everyone agree on the way to insert speaker info inside" according to its issue tracker). I didn't like adding speakers as part of the abstract - it looks worse.

### But won't it be too complicated to change?

It really is easy to write. An `event` needs only `id` OR `guid`, `title`, `start`, `duration`; it's good to also add `abstract` (a.k.a `description`), and `person`(s). There are a few others that I didn't use (`subtitle`, `url`, etc.).

As XML, it can be also nicely indented and separated into visual blocks. Which can be moved around, or have their times changed, now that all events have their guids (I used random UUID v4).

BTW, I used Fosdem and Froscon schedules as templates :heart: 

### Is something missing?

I think we also need a Markdown / HTML version of the schedule - I will propose that separately in near future (maybe with an automated conversion).

Please take a look at your abstracts.

Please think about the general timing of things.

I didn't know how to create paragraphs in the abstracts. An empty line, combined with indentation, caused Giggity to display text as source coede, far from ideal.

### CODE REVIEW

**Comment by mslw** [comment]
Remaining TODOs:

- [ ] add FH's talk
- [ ] adjust time of lightning talks
- [ ] break up MKB and PK's talk into two talks (requested) if feasible
- [ ] take a look and see if the schedule makes sense

Stretch goals:

- [ ] xml to md conversion to have the schedule as text on the website, with easy updates

File: static/sched/distribits2024.xml [hunk 1]
```diff
@@ -0,0 +1,805 @@
...
+    <room guid="" name="Seminar room">
+
+      <event guid="a15745bd-fa3b-4b84-afb1-ff3854b7bd09">
+        <start>08:30</start>
+        <duration>00:30</duration>
+        <title>Coffee</title>
+        <abstract>
+          Coffee time!
+        </abstract>
+      </event>
```
Comment (adswa) [inline static/sched/distribits2024.xml:734]: We could also add a third room to this day, the "lounge" on the third floor. This is where we would have coffee

File: static/sched/distribits2024.xml [hunk 2]
```diff
@@ -0,0 +1,805 @@
...
+        <person></person>
+      </persons>
+    </event>
+
+    </room>
+  </day>
+
+  <day index="3" date="2024-04-06">
+    <room guid="" name="Seminar room">
+
```
[the remark this answers is not in the document]
Comment (mslw) [reply static/sched/distribits2024.xml:726]: With Giggity, the rooms are logical groupings (the views I know display events per room) as much as location hints, so I don't think we need to duplicate the events. But I agree with updating the name - as well as filling in the missing guid.

**Comment by mslw** [comment]
Thank you for your feedback so far!

I think this can be merged, but I'll leave clicking the button to somebody else. Feel free to make any further changes if needed.

Once merged, the URL for giggity (which can be communicated to the outside, and is also included in the schedule page) will be https://distribits.live/sched/distribits2024.xml

One more note: the codespell error is about abstracts that I copied from registration form, I hesitated but I think it would be ok to change the spelling?

File: static/sched/distribits2024.xml [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+      <title>Questions and panel discussion</title>
+      <abstract>
+        Questions and panel discussion
+      </abstract>
+    </event>
+
+    <event guid="4fc2a76d-13be-41fc-bc09-cfcb170ad4ab">
+      <start>13:50</start>
+      <duration>00:20</duration>
+      <title>Reproducable and replicable data science in the presence of patient confidentiality concerns by utilizing git-annex and the Data Science Orchestrator</title>
```
Comment (adswa) [inline static/sched/distribits2024.xml:636]: I'd say we correct the spelling mistakes :) 

```suggestion
      <title>Reproducible and replicable data science in the presence of patient confidentiality concerns by utilizing git-annex and the Data Science Orchestrator</title>
```

File: static/sched/distribits2024.xml [hunk 4]
```diff
@@ -0,0 +1,804 @@
...
+        personally identifiable data, or pseudonymized patient data
+        that carries an inherent risk of depseudonymization.
+        In the project "Data Science Orchestrator", we propose an
+        organizational framework for ethically chaperoning and
+        risk-managing such projects while they are under way, and a
+        software stack that helps in this task. At the same time this
+        software stack will provide an audit trail across the project
+        that is verifyable even by external scientists without access
+        to the raw data, while keeping the option for future
+        reproducability studies and replicability studies open. This
```
Comment (adswa) [inline static/sched/distribits2024.xml:651]:
```suggestion
        reproducibility studies and replicability studies open. This
```

**Pull request merged**